AddressSpace/include/ASNodeQueries.h

```diff
@@ -148,7 +148,7 @@ namespace AddressSpace
         {
             unsigned int nMatched=0;
 #ifdef BACKEND_OPEN62541
-      BOOST_FOREACH( const UaNode::ReferencedTarget &target, *(startNode->referencedTargets()) )
+    for(const UaNode::ReferencedTarget &target : *(startNode->referencedTargets()) )
```

Comment:
not part of the feature i suppose. please clean up.

Reply:
Weirdly it is - I removed BOOST_FOREACH from Meta, which depends on AddressSpace. Without this change ASNodeQueries fails to compile. Weirder still, works on linux but not on windows.

So, I changed to use C++11 range based for. Result: happy in both platforms.

Reply:
I can conditionally accept this change, I suppose the problem is that simply ASNodeQueries.h doesn't  include boost_foreach.hpp which it should.
For future I'd strongly prefer to avoid this type of entangled fixes though.

Reply:
OK. The choice was
a) add include for boost
b) replace boost with std C++11

Went (b)